lib/srv/desktop/rdp/rdpclient/src/rdpdr/mod.rs

```diff
@@ -32,16 +32,31 @@ pub use consts::CHANNEL_NAME;
 /// https://winprotocoldoc.blob.core.windows.net/productionwindowsarchives/MS-RDPEFS/%5bMS-RDPEFS%5d.pdf
 ///
 /// This client only supports a single smartcard device.
+#[allow(dead_code)]
```

Comment:
Is this necessary?

Reply:
Yes, or else it complains about `allow_directory_sharing` not being used.

Reply:
But it is used in the debug logging. Oh well. 

Please try to remove this if/when it becomes possible.